src/resources/postcss/tribe-events-v2-block-editor-reskin.pcss

```diff
@@ -0,0 +1,42 @@
+/**
+ * Block editor event single v2 reskin
+ *
+ * @since TBD
+ */
+
+/* = Website block view styles
+=============================================*/
+.tribe-block__event-website {
```

Comment:
this entire file is in spaces, use tabs.

Reply:
should be fixed